### PR TITLE
[CL-218] Add new icons for extension refresh

### PR DESCRIPTION
**Pull request by vleague2**
## Type of change

```
- [ ] Bug fix
- [x] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

This PR adds a couple of new icons that will be used in the extension refresh project. 

There was also a small change to the puzzle icon, which is intentional by design.

## Screenshots

See the "Storybook Publish" action for a link to Storybook, where you can view the Icons page to see the icons in action.

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: libs/components/src/stories/icons.mdx [hunk 1]
```diff
@@ -138,6 +143,7 @@ or an options menu icon.
 | <i class="bwi bwi-cut"></i>               | bwi-cut               | cut or omit actions                            |
 | <i class="bwi bwi-dashboard"></i>         | bwi-dashboard         | statuses or dashboard views                    |
 | <i class="bwi bwi-desktop"></i>           | bwi-desktop           | desktop client                                 |
+| <i class="bwi bwi-desktop-alt"></i>       | bwi-desktop-alt       | desktop client                                 |
```
Comment (willmartian) [inline libs/components/src/stories/icons.mdx:146]: `bwi-desktop-alt` and `bwi-browser-alt` are the same icon, is this intended? Also, do we need extra variants, or can we just replace the existing ones?

File: libs/components/src/stories/icons.mdx [hunk 2]
```diff
@@ -41,6 +41,7 @@ or an options menu icon.
 | <i class="bwi bwi-sticky-note"></i> | bwi-sticky-note | secure note item type                               |
 | <i class="bwi bwi-users"></i>       | bwi-users       | user group                                          |
 | <i class="bwi bwi-vault"></i>       | bwi-vault       | general vault                                       |
+| <i class="bwi bwi-vault-f"></i>     | bwi-vault-f     | general vault                                       |
```
Comment (willmartian) [inline libs/components/src/stories/icons.mdx:44]: ~Could we updated places where we use `-f` to use `-filled` instead? That is more clear to me.~

Nvm, I see we are already using `-f` elsewhere.

File: libs/components/src/stories/icons.mdx [hunk 3]
```diff
@@ -153,7 +159,9 @@ or an options menu icon.
 | <i class="bwi bwi-lightbulb"></i>         | bwi-lightbulb         | -                                              |
 | <i class="bwi bwi-link"></i>              | bwi-link              | link action                                    |
 | <i class="bwi bwi-mobile"></i>            | bwi-mobile            | mobile client                                  |
+| <i class="bwi bwi-mobile-alt"></i>        | bwi-mobile-alt        | mobile client                                  |
```
[the remark this answers is not in the document]
Comment (danielleflinn) [reply libs/components/src/stories/icons.mdx:162]: I'm fine if we leave all the "alt" device icons out as I believe these were added to the shared font file by Marketing.

File: libs/components/src/stories/icons.mdx (hunk 1 above)
Comment (danielleflinn) [reply libs/components/src/stories/icons.mdx:146]: Looks like this was a bug with the font file export; if we decide to keep the "alt" icons, let me know and I can update the file.

Comment (vleague2) [reply libs/components/src/stories/icons.mdx:146]: Lol how did I not notice they look the same 🤦

File: libs/components/src/stories/icons.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (willmartian) [reply libs/components/src/stories/icons.mdx:162]: Talked sync about this

**Review by willmartian — APPROVED** [review]
LGTM!

**Pull request merged**